starter-code/api/models/rides.js

```diff
@@ -21,4 +21,14 @@ const rideSchema = new mongoose.Schema({
     seats_available: Number
 })
 
-module.exports = mongoose.model('Rides', rideSchema)
\ No newline at end of file
+const testRide = new mongoose.Schema({
+    id: String,
+    name: String,
+    startLocation: String,
+    endLocation: String,
+    dayLeave: String,
+    timeLeave: String
+})
```

Comment:
I'll most likely change/integrate these into one schema once I write the backend API